connect/transforms/src/test/java/org/apache/kafka/connect/transforms/MaskFieldTest.java

```diff
@@ -153,4 +179,56 @@ public void withSchema() {
         assertEquals(Collections.emptyMap(), updatedValue.get("map"));
     }
 
+    @Test
+    public void schemalessWithReplacement() {
```

Comment:
Nit: for better or worse, our convention is to use `test` as a prefix for the test methods, so this would be:
```suggestion
    public void testSchemalessWithReplacement() {
```